app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_branch_mixin.py

```diff
@@ -55,162 +49,166 @@ def test_serializer_replace_branches(self):
         self.assertTrue(serializer.is_valid())
         serializer.save()
         experiment = NimbusExperiment.objects.get(id=experiment.id)
+        self.assertEqual(experiment.feature_configs.get(), feature_config)
 
-        for key, val in reference_branch.items():
-            self.assertEqual(getattr(experiment.reference_branch, key), val)
-
-        for branch_data in treatment_branches:
-            branch = experiment.branches.get(name=branch_data["name"])
-            for key, val in branch_data.items():
-                self.assertEqual(getattr(branch, key), val)
+        self.assertEqual(experiment.reference_branch.name, "control")
```

Comment:
Hmm, could be nice to source this "control" from `reference_branch["name"]` just to retain some of the DRY-ness of the previous version. But, not a win on the verbosity side of things to change all these literal values to references

Reply:
Yeah I've been waffling back and forth on this recently as I work through tests, whether to reference the value or hard code it in the assertion.  I can kinda convince myself of either but I was trying to lean towards the explicit side here just to see if that makes me feel better about the tests.  Sometimes you can use references in assertions that are wrong on both sides and then miss it and then the test passes even though it's broken and you miss it because the reference looks right.  I dunno, something to think about 🤷‍♀️